### PR TITLE
DOC: Adding Cartopy v0.19 documentation

### PR DESCRIPTION
**Pull request by greglucas**
I followed the instructions in the README and just added all files to the git tracking.

Closes SciTools/cartopy#1764

### CODE REVIEW

**Comment by greglucas** [comment]
@QuLogic, do you have commit rights here?

File: cartopy/docs/v0.19/.buildinfo [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
```
Comment (QuLogic) [inline cartopy/docs/v0.19/.buildinfo:1]: Please delete this file.

Comment (greglucas) [reply cartopy/docs/v0.19/.buildinfo]: Done, any others hanging around too? I just pulled straight from the CircleCI doc build.

Comment (QuLogic) [reply cartopy/docs/v0.19/.buildinfo]: No, I think that's all.

**Pull request merged**